std/parallelism.d

```diff
@@ -2864,13 +2864,13 @@ public:
             }
         }
 
-        ref T opIndex(size_t index)
+        ref T opIndex(size_t index) const
```

Comment:
I don't understand. A `const` member function is able to return a mutable reference because the member function itself isn't modifying the struct.

And, if the element type is immutable/const then `T` will be immutable/cons as well